docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx

```diff
@@ -1,13 +1,168 @@
 ---
-title: 'VS Code Plugin'
-sidebar_label: 'VS Code Plugin'
+title: "VS Code Plugin (Genesis UI Generator)"
+sidebar_label: "VS Code Plugin"
 id: development-environment-vs-code-plugin
-keywords: [vs code, vscode, plugin]
+keywords: [web, tooling, code snippets, vscode, extension, ai, local]
 tags:
-- vs code
-- vscode
-- plugin
+  - web
+  - tooling
+  - code snippets
+  - vs code
+  - vscode
+  - plugin
+  - extension
+  - ai
 sidebar_position: 5
 ---
 
-Coming soon...
+The Genesis UI Generator is a Visual Studio Code plugin designed to enhance the development experience on the Genesis Platform.
+
+You provide an image of the UI you want to create, and the Genesis UI Generator creates the code for a working UI, along with the files that enable you to add it to a new or existing application. This can be a major boost to productivity.
+
+## Requirements
+
+- Visual Studio Code version 1.88.0 or greater.
+- Install [literally-html](https://marketplace.visualstudio.com/items?itemName=webreflection.literally-html), which provides syntax highlighting and IntelliSense for html inside JavaScript and TypeScript tagged template strings.
+
+## Installation
+
+1. To install the Genesis UI Generator, visit [Genesis UI Generator on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=GenesisGlobal.genesis-ui-generator). Click on **Install** or search for `Genesis UI Generator` in the extensions panel in VS Code.
+
+2. After installing the extension, the Genesis icon will be visible in the VS Code side menu on the left.
+
+![Genesis UI Generator Icon](/img/vscode-extension/icon-extension.png)
+
+## Configuration
+
+- Open your Genesis project (root folder) in VS Code before starting to use the extension features.
+- Ensure that the source folder path (`src`) is correctly configured in the VS Code extension's settings.
```

Comment:
I know this is quite obvious for us but it's maybe worth explaining how to do that in detail as our target audiance are people who probably don't feel very comfortable configuring tools

Reply:
makes sense, will add more details on it.

Reply:
Done. updated